ci_framework/roles/edpm_prepare/tasks/main.yml

```diff
@@ -115,6 +115,18 @@
   ansible.builtin.include_role:
     name: set_openstack_containers
 
+- name: Patch ansible runner image temprorily
+  when: not cifmw_edpm_prepare_update_os_containers | bool
```

Comment:
we need another condition to skip this, since in downstream we don't want use os containers and also don't want to use upstream ansiblee-runner. So I think that this need a especific skip flag